public/libs/customLinksModal.js

```diff
@@ -0,0 +1,58 @@
+CustomLinks_removeEntityFromSelectionList = function (relationship) {
```

Comment:
Change names to be valid (camelCase). Use `function <name>(relationship) {}` syntax.